### ISSUE TITLE
I CANT UNINSTALL!

### ISSUE THREAD

**Issue by otaviomuraca** [dunovank/jupyter-themes#452]
Please somebody help to uninstall the jupyter-themes!

I tried used some commands, and reinstall Anaconda and delete all cache of my macbook but nothing working! 

I need to back this normal, I lost my day  of work trying to fix it. 

I really need help please!

**Comment by dunovank** [comment]
hey @otaviomuraca sorry to hear you're having trouble. Can you paste the uninstall commands and any command line outputs?

**Comment by otaviomuraca** [comment]
Hi @dunovank 

I tried using the following codes:
 - jt -r
 - pip uninstall jupyterthemes
 
This is return: WARNING: Skipping jupyterthemes as it is not installed.  But the theme still on.

**Comment by dunovank** [comment]
So if the `pip uninstall` command returns `WARNING: Skipping jupyterthemes as it is not installed. But the theme still on.` then my guess is that you're in a separate virtual environment from the one where you originally installed the `jt` package. Is it possible something like this is going on? 

If you're not sure and just want to uninstall, first close down any jupyter kernels currently running (fully shut down jupyter).  Next, open up a fresh terminal and check to see if `~/.jupyter/custom` exists by running:

```sh
ls ~/.jupyter | grep custom
```

If it exists, then you will see the word `custom` printed in the output. If so, recursively (`-r`) delete that folder & its contents by running:

```sh
 rm -rf ~/.jupyter/custom
```

Finally, open up a new jupyter session. If you still see the jupyterthemes UI style, clear the browser cache, close down jupyter, terminate the running jupyter terminal job, close down your browser and re-start a fresh jupyter session. If you wouldn't mind trying these steps and reporting the results back here I'd appreciate it.

**Comment by otaviomuraca** [comment]
@dunovank,

I performed the procedure and it actually generated a `custom`.

When trying to open Jupyter again, it appeared an **installation error** and requested a new installation:

**Error: EnvironmentLocationNotFound: Not a conda environment: /opt/anaconda3**

I installed Jupyter and it returned to normal. **It was reset successfully.**

Thank you so much for the help **I really appreciate it**.

I will follow your project for testing again in the future.

Good luck! ;)
